src/ensembl/src/content/app/species-selector/components/species-selector-app-bar/SpeciesSelectorAppBar.tsx

```diff
@@ -66,15 +62,22 @@ export const SpeciesSelectorAppBar = (props: Props) => {
 };
 
 const SelectedSpeciesList = (props: Props) => {
+  const showSpeciesHomepage = (species: CommittedItem) => {
+    const speciesPageUrl = urlFor.speciesPage({
+      genomeId: species.genome_id
+    });
+
+    props.push(speciesPageUrl);
+  };
+
   const shouldLinkToGenomeBrowser =
     props.selectedSpecies.filter(({ isEnabled }) => isEnabled).length > 0;
 
   const selectedSpecies = props.selectedSpecies.map((species) => (
     <SelectedSpecies
       key={species.genome_id}
       species={species}
-      onToggleUse={props.toggleSpeciesUse}
-      onRemove={props.onSpeciesDelete}
+      onClick={() => showSpeciesHomepage(species)}
```

Comment:
`SelectedSpecies` component already passes the genome id to the click handler; so there's no need to pass the whole species around. `onClick={showSpeciesHomepage}` should be sufficient.